src/ui/src/wds/WdsJsonViewerValue.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+	<span class="WdsJsonViewerValue">{{ dataFormatted }}</span>
+</template>
+
+<script setup lang="ts">
+import { PropType, computed } from "vue";
+import type { JSONValue } from "./WdsJsonViewer.utils";
+
+const props = defineProps({
+	data: { type: Object as PropType<JSONValue>, required: true },
+});
+
+const dataFormatted = computed(() => JSON.stringify(props.data));
+</script>
+
+<style scoped>
+.WdsJsonViewerValue {
+	font-family: monospace;
+	color: var(--jsonViewerValueColor, orangered);
```

Comment:
Let's not introduce colors, let's use normal/primaryColor and we can always come back to it and discuss with designers.

Reply:
or secondary text color is fine too.

Reply:
Do you mean:

1. using the secondary color as default like
```suggestion
	color: var(--jsonViewerValueColor, var(--secondaryTextColor));
```
2. or completly remove the possibilty to customize the color and always use secondary color 
```suggestion
	color: var(--secondaryTextColor);
```

Reply:
Let's go for option 2 but make sure we're adding the option to configure `secondaryTextColor`.